packages/appstore/src/components/wallet-button/__tests__/wallet-button.spec.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import { mockStore, StoreProvider } from '@deriv/stores';
+import WalletButton from '..';
+
+const mockedRootStore = mockStore({});
+
+describe('<WalletButton />', () => {
+    const button = {
+        name: 'Transfer',
+        text: 'Transfer',
+        icon: 'IcAccountTransfer',
+        action: () => {
+            return true;
+        },
+    } as const;
+
+    it('Should render right text', () => {
+        render(
+            <StoreProvider store={mockedRootStore}>
+                <WalletButton button={button} />
+            </StoreProvider>
+        );
+        const button_text = screen.queryByText('Transfer');
+
+        expect(button_text).toBeInTheDocument();
```

Comment:
can we use getByText instead?

```suggestion
        expect(screen.getByText('Transfer')).toBeInTheDocument();
```

Reply:
Updated.